src/serlo-editor/plugins/serlo-table/editor.tsx

```diff
@@ -147,10 +138,9 @@ export function SerloTableEditor(props: SerloTableProps) {
             <div
               key={colIndex}
               tabIndex={0} // capture tab
```

Comment:
```suggestion
```
This can (and should) now also be removed, otherwise it takes two tabs to focus slate.

Reply:
postponed work on table plugin